rts/lra/lra-coordinator-jar/README.md

```diff
@@ -0,0 +1,28 @@
+Enable LRA JWT Security
+=======================
+
+
+Requirements
+------------
+
+* Always name the JWT token **Realm** as: `narayana-lra-jwt`
+
+* Allowed Roles: `{"admin","participant"}`
+
+Before starting LRA-coordinator, we have to first set some System/Environment variable properties, in order to enable LRA-JWT-security layer.
```

Comment:
Can these be MP config properties?

Reply:
It depends on the server on which application is running, if it takes MP config properties into consideration, it would work.

Reply:
Then change this to:
```suggestion
Before starting LRA-coordinator, we have to first set some MicroProfile Config properties, in order to enable LRA-JWT-security layer.
```

Reply:
okay